applications/trilinos_application/custom_factories/trilinos_linear_solver_factory.h

```diff
@@ -119,8 +119,8 @@ typedef LinearSolverFactory<TrilinosSparseSpaceType,  TrilinosLocalSpaceType> Tr
     KratosComponents<TrilinosLinearSolverFactoryType>::Add(name, reference);
 
 
-template<class TComponentType>
-typename KratosComponents<TComponentType>::ComponentsContainerType KratosComponents<TComponentType>::msComponents;
+// template<class TComponentType>
+// typename KratosComponents<TComponentType>::ComponentsContainerType KratosComponents<TComponentType>::msComponents;
```

Comment:
could you please remove and not comment this?
I am usually getting confused when I see stuff like this :)

Reply:
Done!